projects/packages/forms/src/dashboard/inbox/export-modal/google-drive.js

```diff
@@ -51,6 +53,13 @@ const GoogleDriveExport = ( { onExport } ) => {
 			} );
 	}, [ onExport ] );
 
+	const handleConnectClick = useCallback( () => {
+		tracks.recordEvent( 'jetpack_forms_upsell_googledrive_click', {
+			screen: 'form-submissions-inbox',
+		} );
+		pollForConnection();
+	}, [ tracks, pollForConnection ] );
+
```

Comment:
I know this tracking implementation with useAnalytics hook works. I just recently implemented it in another file. But with this code as is, I do not see the Tracks even fire in Tracks Vigilante. For the 'classic' version of the forms screen, there was a similar problem, and it was because the page was redirecting before tracks could fire. But here, even if I completely prevent the redirect, the track event is not firing. Anyone have thoughts on why?

Reply:
This was it. We needed to load the tracking scripts. Thanks!